GETTING-STARTED.md

```diff
@@ -3,12 +3,12 @@ Hello! Thanks for wanting to work on this project :smile:, and I hope that this
 Note that this document is about contributing code for Cuberite.
 
  - If you are looking for usage instructions, see the [user manual](https://book.cuberite.org/) instead.
```

Comment:
Could you change 'user manual' to 'user's manual' here, as that is the preferred name.